### PR TITLE
Document `--cpus` and `--memory` flags for `crc start`

### PR DESCRIPTION
**Pull request by robin-owen**
Also removes outdated information about deleting an existing virtual machine to apply configuration changes.

### CODE REVIEW

**Comment by robin-owen** [comment]
By the way, I'd like to asynchronously update the existing downstream documentation with these changes once this PR is merged. :)

File: docs/source/topics/proc_configuring-the-virtual-machine.adoc [hunk 1]
```diff
@@ -3,43 +3,38 @@
...
 The number of vCPUs to assign must be greater than or equal to the default.
 
+* To start the virtual machine with the desired number of vCPUs:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} start --cpus __<number>__
+----
+
 * To increase the memory available to the virtual machine:
```
Comment (cfergeau) [inline docs/source/topics/proc_configuring-the-virtual-machine.adoc:33]: Maybe also change this to `configure` since you changed it for the vCPUs?

Comment (robin-owen) [reply docs/source/topics/proc_configuring-the-virtual-machine.adoc]: Missed that, thanks! Updated. :)

**Review by cfergeau — APPROVED** [review]

**Pull request merged**